model/span/event.go

```diff
@@ -388,7 +388,7 @@ func (e *Event) Transform(tctx *transform.Context) []beat.Event {
 	}
 
 	// first set the generic metadata
-	tctx.Metadata.SetMinimal(fields)
+	tctx.Metadata.Set(fields)
 
 	// then add event specific information
 	utility.DeepUpdate(fields, "service", e.Service.MinimalFields())
```

Comment:
This should be updated to `utility.DeepUpdate(fields, "service", e.Service.Fields())`

Reply:
good catch :+1